docs/src/tutorials/using_catalyst.md

```diff
@@ -0,0 +1,327 @@
+# Using Catalyst
+In this tutorial we'll provide an introduction to using Catalyst to specify
+chemical reaction networks, and then to solve ODE, jump and SDE models generated
+from them. Let's start by using the Catalyst [`@reaction_network`](@ref) macro
+to specify a simply chemical reaction network; the well-known repressilator.
+
+We first import the basic packages we'll need:
+
+```julia
+# If not already installed, first hit "]" within a Julia REPL. Then type:
+# add DiffEqBase OrdinaryDiffEq StochasticDiffEq DiffEqJump ModelingToolkit Catalyst Plots Latexify 
```

Comment:
we should really just recommend DifferentialEquations, reexport MTK. This should just be:

```julia
using Catalyst, DifferentialEquations, Plots, Latexify
```

Reply:
It used to be that in the old tutorial, but I was finding `using DifferentialEquations` to have become very slow (so much that I haven't used it in quite a while, so perhaps that improved).
In real use I don't think one would generally want to use it would they?

Reply:
on the other hand, saying we're going to `using` 8 packages in the basic tutorial is quite intimidating. But yeah.. FWIW using times are dropping heavily in v1.5 and v1.6.

Reply:
I switched it. I was debating that same point myself but had leaned towards it being better to show what I'd think is the normal use case vs. the easier case. But it is fine either way.